jekyll/_cci2/language-android.md

```diff
@@ -133,12 +147,107 @@ and macOS capabilities. Please check out [this example React Native
 application](https://github.com/CircleCI-Public/circleci-demo-react-native)
 on GitHub for a full example of a React Native project.
 
-## Disabling Pre-Dexing to Improve Build Performance
```

Comment:
It feels odd to name a specific reason for performing an action in a section heading.

Reply:
It isn't odd, this is a bit surprising, so that is why I made the heading this way to be very clear.